tech.json

```diff
@@ -2387,6 +2387,44 @@
                 "Holding back and jump for more than 5 frames will cause Samus to spark horizontally. If the direction switch was exactly one frame late, Samus will land on the slope but no spark will occur."
               ]
             },
+            {
+              "name": "canRModeCrystalFlashInterrupt",
+              "techRequires": [
+                "canRMode",
+                {"tech": "canCrystalFlash"},
+                "canCarryFlashSuit"
+              ],
+              "otherRequires": [
+                {"resourceAtMost": [{"type": "RegularEnergy", "count": 50}]},
+                {"resourceAtMost": [{"type": "ReserveEnergy", "count": 0}]},
+                {"or": [
+                  {"disableEquipment": "ETank"},
+                  "canRiskPermanentLossOfAccess"
```

Comment:
I think the `canRiskPermanentLossOfAccess` option would need additional requirements to express the low amount of E-Tanks. Might be better to drop it for now?